test/unit/plugins/rest_spec.js

```diff
@@ -79,6 +82,8 @@ describe("The Rest plugin", function () {
 					expect(response.headers.location, "location").to.match(location);
 					expect(response.payload.ami, "ami").to.equal(VALID_AMI);
 					expect(response.payload.type, "type").to.equal(VALID_TYPE);
+					/*Required for GET Requests*/
+					VALID_INSTANCE_ID = response.payload.id;
```

Comment:
Doing this starts to mix state between test cases. It's better practice to create any resources you need to test functionality within a single test block. So, when you want to test fetching an instance, have that test create its own instance before making the GET request.